### PR TITLE
PMM-2119 Add warning message for pmm dashboards

### PR DESCRIPTION
**Pull request by darly-s**

### CODE REVIEW

File: src/app/query-profile/query-profile.component.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
         PMM documentation.
     </a>
 </div>
+<div *ngIf="isAll" class="alert alert-warning w-75 mx-auto text-center wrap" role="alert">
+  Please select  the  Database Host from the  Host selector on the top of the page
```
Comment (askomorokhov) [inline src/app/query-profile/query-profile.component.html]: Please clean up spaces.

File: src/app/settings/settings.component.html [hunk 2]
```diff
@@ -1,3 +1,6 @@
+<div *ngIf="isAll" class="alert alert-warning w-75 mx-auto text-center wrap" role="alert">
+  Please select  the  Database Host from the  Host selector on the top of the page
```
Comment (askomorokhov) [inline src/app/settings/settings.component.html]: same here

File: src/app/summary/summary.component.html [hunk 3]
```diff
@@ -1,3 +1,6 @@
+<div *ngIf="isAll" class="alert alert-warning w-75 mx-auto text-center wrap" role="alert">
+  Please select  the  Database Host from the  Host selector on the top of the page
```
Comment (askomorokhov) [inline src/app/summary/summary.component.html]: and here.

**Comment by darly-s** [comment]
Should be closed due to create new one (https://github.com/percona/qan-app/pull/96).

**Pull request closed without merging**